docs/doxygen/ie_docs.xml

```diff
@@ -94,6 +94,7 @@
                 <!-- IR -->
                 <tab type="usergroup" title="Intermediate Representation and Operations Sets" url="@ref openvino_docs_MO_DG_IR_and_opsets">
                     <tab type="usergroup" title="Available Operations Sets" url="@ref openvino_docs_ops_opset">
+                        <tab type="user" title="opset5 Specification" url="@ref openvino_docs_ops_opset5"/>
```

Comment:
Add a link to the `docs/ops/sequence/RNNSequence_5.md` in the section below.

Reply:
Updated, please verify.